### ISSUE TITLE
Idea: Ishkendrin's Battlesphere should persist like other summons

### ISSUE THREAD

**Issue by eliotn** [jeremygurr/dcssca#635]
The battle sphere spell essentially summons something, so why shouldn't it persist like other summons?  It could persist according to spellpower.

**Comment by jeremygurr** [comment]
Yeah it should. There are a number of spells like this that need some polishing in the new system.

**Comment by yrmvgh** [comment]
This makes me sound crazy, because I like the perma summons (and I might have been the person who suggested permasummons way back when?). But I don't think Battlesphere should be perma, because it's not a Summon spell, and currently Charms aren't perma.

And for charms, many of them make less sense at infinite duration  - and duration is often the only scaling factor. Whereas Summons mostly had sufficient duration to last "the whole fight", except Hydras which were uniquely powerful + shortlived.

But anyway, long story short, I like that Battlesphere is a "Super Searing Ray" that fires on its own without requiring you to stand still, it's pretty different from a normal summon. In a previous, very-related thread, we came up with [some other ideas](https://github.com/jeremygurr/dcssca/issues/336) for **automatically recasting** Bsphere, rather than it lasting forever.

**Comment by yrmvgh** [comment]
And that thread ain't dead, so @eliotn, feel free to weigh in here or on that issue there, even if just to say "Listen Y, you smell and I still think Bsphere should be perma." :wink:

**Comment by jeremygurr** [comment]
I don't see a reason why charms should not be permanent. It reduces tediousness, and the charm can be tested against spell failure like transmutations. The effectiveness of the spell should be based on spell power, but how long it lasts should be limited by spell failure. Spell cost is much higher than normal, since it is semi-permanent.
